src/local-history/local-history-manager.ts

```diff
@@ -136,32 +134,26 @@ export class LocalHistoryManager {
     public async saveActiveEditorContext(document: vscode.TextDocument): Promise<void> {
```

Comment:
@vince-fugnitto Should I change this method name to `saveEditorContext` so that it is more concise?